praw/util/__init__.py

```diff
@@ -1,21 +1,4 @@
 """Package imports for utilities."""
 
-import re
-
-__all__ = ("cache", "camel_to_snake", "snake_case_keys")
```

Comment:
Can we still have `__all__` exist as a tuple that lists all importable names?

Reply:
Sure, but I wonder why this isn’t done throughout all of the other modules

Reply:
Ah, fair enough. Since we don't do that elsewhere, we don't need it here so long as tests pass.

Reply:
@jarhill0 I think it’s still good coding practice to do it so I’ll push out PRs tomorrow that do the same for the other modules.

Reply:
Actually, I respectfully disagree. Using `__all__` adds one more thing that has to be updated and might get out of sync. I prefer not using it, and I'll merge this PR once it's removed. Sorry for the confusing messaging on my part.